flexible_event_config.md

```diff
@@ -427,6 +427,10 @@ is dropped.
 
 We encourage developers to suggest different use cases they may have for this API extension, and we will update this explainer with sample configurations for those use cases.
 
+## Attribution Rate Limit
+
+Attribution rate-limit behavior will be based on the numbers of reports generated rather than number of triggers that lead to report generation. For example: The 1st report in the [Reporting trigger value buckets](#reporting-trigger-value-buckets) example will be counted as 1 contribution towards the [max attribution per rate-limit window](https://wicg.github.io/attribution-reporting-api/#max-attributions-per-rate-limit-window) limit despite it being a result of contributions from 3 triggers.
```

Comment:
If Full Flex is accepted, would it make more sense to rename this limit to reflect that it is report-based, rather than attribution-based? The current naming is confusing.

Reply:
Yes, I think so. Maybe something on the lines of max reported-attributions, max reported-conversions or just max reports.

Reply:
I'd support renaming it already.

Reply:
One caveat with "max reports" is the fake event-level reports, for which we only count 1 regardless of how many fake reports it creates.

Reply:
Does `max reported-attributions per rate limit window` sound good to everyone? @linnan-github , @apasel422 & @giladbarkan-github

Reply:
Fine with me. Renaming it all here could be a big change, though. I assume we'd do it after?

Reply:
Ack. Doing it in a follow-up sounds good to me.